### PR TITLE
docs: several enhancements on docs including index and installation

### PR DESCRIPTION
**Pull request by d0choa**

### CODE REVIEW

File: pyproject.toml
Comment (DSuveges) [inline pyproject.toml:6]: Is the genetics team part of the "core team"? If not, we should also mention them.

**Review by DSuveges — APPROVED** [review]
I had only one quesiton, which is also not tightly related to this PR.

File: pyproject.toml
Comment (d0choa) [reply pyproject.toml:6]: they are

**Comment by d0choa** [comment]
there is another branch from @addramir. I'll consolidate before merging

**Pull request merged**
